src/main/java/com/sysc4806app/model/ProductChain.java

```diff
@@ -0,0 +1,78 @@
+package com.sysc4806app.model;
+
+/**
+ * @version milestone 1
+ *
+ * A product chain. Can be used to categorize products.
+ */
+public enum ProductChain implements PathVariableEnum {
+    SWY("Subway", "sway", ProductChainTag.SANDWICHES),
```

Comment:
idk if limiting the possible chains to a hardcoded enum is the best idea

Reply:
Like what if a user want to review some other fast food place that isn't on the list

Reply:
> Like what if a user want to review some other fast food place that isn't on the list

Allowing users to make food chains and categories also isn't a great option. We've been discussing what the plan is for the app whether we allow users to create whatever they wish or limiting the categorization to accepted values, and it depends on our intention.